### PR TITLE
Add fake test component

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- add external solution test fake component

**Related issue(s)**

### CODE REVIEW

File: components/fake-external-test-component/internal/configuration/service.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+type Service struct {
+	configLogs map[string]model.ConfigurationChange
+}
+
+func NewService() *Service {
+	return &Service{configLogs: make(map[string]model.ConfigurationChange)}
+}
+func (s *Service) Save(change model.ConfigurationChange) (string, error) {
+	id := uuid.New().String()
+	id = "0ef99922-c493-4ccf-b8de-1778ab84ca3a"
```
Comment (dbadura) [inline components/fake-external-test-component/internal/configuration/service.go]: To fix

File: components/fake-external-test-component/internal/security/service.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+type Service struct {
+	configLogs map[string]model.SecuritEvent
+}
+
+func NewService() *Service {
+	return &Service{configLogs: make(map[string]model.SecuritEvent)}
+}
+func (s *Service) Save(change model.SecuritEvent) (string, error) {
+	id := uuid.New().String()
+	id = "0ef99922-c493-4ccf-b8de-1778ab84ca3a"
```
Comment (dbadura) [inline components/fake-external-test-component/internal/security/service.go]: to fix

File: components/fake-external-test-component/pkg/model/securityevent.go [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package model
+
+import (
+	"net"
+	"time"
+)
+
+type SecuritEvent struct {
```
Comment (dbadura) [inline components/fake-external-test-component/pkg/model/securityevent.go]: SecutiyEvent

File: components/fake-external-test-component/internal/configuration/handler.go [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+const (
+	HeaderContentTypeKey   = "Content-Type"
+	HeaderContentTypeValue = "application/json;charset=UTF-8"
+)
+
+func (h *ConfigChangeHandler) Save(writer http.ResponseWriter, req *http.Request) {
+	defer h.closeBody(req)
+
+	var log model.ConfigurationChange
```
Comment (kjaksik) [inline components/fake-external-test-component/internal/configuration/handler.go]: That variable name might be misleading

**Comment by dbadura** [comment]
Will be reopen if we implement auditlog tests.

**Pull request closed without merging**